src/commands.ts

```diff
@@ -13,12 +13,12 @@ import {
     Uri,
     ViewColumn
 } from 'vscode';
-import { IShowDataViewerFromVariablePanel } from '../../messageTypes';
```

Comment:
There are 59 file modified but most of them are import path changes and file movement.